src/main/web/CMakeLists.txt

```diff
@@ -1,26 +1,22 @@
-set(SCRIPT "${CMAKE_CURRENT_SOURCE_DIR}/scripts/gen_embedded.py")
+set(SCRIPT "${PROJECT_SOURCE_DIR}/scripts/gen_embedded.py")
 set(PYTHON "python2")
 
+
 set(EMBEDDED_FILES 
-  _404.png
-  _error.css
-  _error.html
-  favicon.ico
-  _jquery.min.js
-  _seasocks.css
-  _stats.html
+    ${CMAKE_CURRENT_SOURCE_DIR}/_404.png
```

Comment:
Oh right - since these are passed to a script they're not generated correctly by cmake itself. I keep removing these prefixes in add_library where they actually are not needed.